ui/options_plugins.ui

```diff
@@ -98,7 +98,7 @@
             </sizepolicy>
            </property>
            <property name="text">
-            <string>Download plugins</string>
+            <string>Reload available plugins</string>
```

Comment:
I think this is ambiguous - it could mean both "re-import all plugins" or "reload the information about available and installed plugins".